### ISSUE TITLE
custom colums in template stopped working after update to v1.26.1 from v1.20.1

### ISSUE THREAD

**Issue by synbitz** [fboender/ansible-cmdb#158]
Hello, 

I recently updated to version 1.26.1. I had built a "custom" template that was just the html_fancy template with some extra columns that show custom facts. This worked fine but when I edited the html_fancy template that came with 1.26.1 it does not work anymore.  I hope you understand what I mean and I hope you can help me 

So in html_fancy_defs.html I added : 

    
```

{"title": "Base",          "id": "base",          "func": col_base,           "sType": "string", "visible": True},
{"title": "Integrator",    "id": "integrator",    "func": col_integrator,     "sType": "string", "visible": True},


```
in the column definitions and a bit lower I had defined them like : 

```
`<%def name="col_base(host, **kwargs)">
  % if 'base' in host['ansible_facts']['ansible_local']:
    ${jsonxs(host, 'ansible_facts.ansible_local.base.Base', default='')}
  % endif
</%def>

<%def name="col_integrator(host, **kwargs)">
  % if 'integrator' in host['ansible_facts']['ansible_local']:
    ${jsonxs(host, 'ansible_facts.ansible_local.integrator.Integrator', default='')}
  % endif
 
</%def>
```
`

this worked fine in 1.20 but now in 1.26.1 I am not getting the custome colums in the overview anymore..

Hopefully I can get these back in there without downgrading, I really need the bug fixes and extra functions in 1.26

thank you

**Comment by fboender** [comment]
In order to get this to work, I had to change the functions (`<%def`) to:

    <%def name="col_base(host, **kwargs)">
      ${jsonxs(host, 'ansible_facts.ansible_local.base.Base', default='')}
    </%def>

    <%def name="col_integrator(host, **kwargs)">
      ${jsonxs(host, 'ansible_facts.ansible_local.integrator.Integrator', default='')}
    </%def>

Other than that, it seems to work for me. Are you getting an error or just empty results for the fields?

One thing you can try is to click the 'Clear settings' button at the top-right of ansible-cmdb. Sometimes, column toggle settings get confused when columns change.

**Comment by synbitz** [comment]
Hello. I'm so sorry to have bothered you with this non issue. I had edited the package files at the wrong location. I have ansible-cmdb running in a virtual environment and in the LIB directory there were two directories, one named ansiblecmdb and one named python3.5. I edited the template files in the ansiblecmd directory instead of going in the python3.5/dist-packages , where the actual files are that were being used...

so an issue at my side so sorry to have bothered you, thank you for the quick reply and also thank you for this tool we are making good use of it and hope to be able to continue to use it.
